Saas/Install-BcAppFromAppSource.ps1

```diff
@@ -1,4 +1,4 @@
-﻿<#
+<#

```

Comment:
there seem to be a change in UTF8 BOM here???

Reply:
Ops, I use GitHub to update the repository. I'll fixed that.